Error calling the CLI

## Description

After installing the CLI and trying to run, I got:

```
$ pnpm codeowners-flow     
node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/path/to/node_modules/.pnpm/@codeowners-flow+cli@0.1.1/node_modules/@codeowners-flow/cli/dist/index.js' imported from /path/to/node_modules/.pnpm/@codeowners-flow+cli@0.1.1/node_modules/@codeowners-flow/cli/bin/index.js
    at new NodeError (node:internal/errors:371:5)
    at finalizeResolution (node:internal/modules/esm/resolve:418:11)
    at moduleResolve (node:internal/modules/esm/resolve:981:10)
    at defaultResolve (node:internal/modules/esm/resolve:1078:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:530:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:251:18)
    at ESMLoader.import (node:internal/modules/esm/loader:332:22)
    at importModuleDynamically (node:internal/modules/esm/translators:106:35)
    at importModuleDynamicallyCallback (node:internal/process/esm_loader:35:14)
    at file:///path/to/node_modules/.pnpm/@codeowners-flow+cli@0.1.1/node_modules/@codeowners-flow/cli/bin/index.js:4:1 {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

Checking the node_modules, it seems the `dist` folder isn't there 🫠

![CleanShot 2023-08-16 at 09 36 11](https://github.com/raulfdm/codeowners-flow/assets/12464600/0515fe5a-3aed-4612-bf64-83094d5c6c96)

--- a/.changeset/popular-kids-rest.md
+++ b/.changeset/popular-kids-rest.md
@@ -1,0 +1,5 @@
+---
+'@codeowners-flow/cli': patch
+---
+
+publish dist folder

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -40,7 +40,8 @@
     }
   },
   "scripts": {
-    "prepare": "pnpm turbo build --filter codeowners-flow --output-logs errors-only",
+    "prepare": "pnpm turbo build --filter cli --output-logs errors-only",
+    "prepack": "pnpm turbo build --filter cli --output-logs errors-only",
     "build": "vite build",
     "dev": "tsx src/index.ts",
     "lint": "eslint src",

